bower.json

```diff
@@ -0,0 +1,20 @@
+{
+  "name": "vui-table",
+  "main": "table.scss",
+  "license": "Apache-2.0",
+  "ignore": [
+    "node_modules",
```

Comment:
Bower points to repo, so we only need to ignore stuff that is in GitHub.  I don't think it's necessary to specify node_modules and bower_components here.